views/templates/hook/ps_googleanalytics.tpl

```diff
@@ -17,39 +17,57 @@
  * International Registered Trademark & Property of PrestaShop SA
  *}
 
+{if $isV4Enabled}
+  <script async src="https://www.googletagmanager.com/gtag/js?id={$gaAccountId|escape:'htmlall':'UTF-8'}"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    {literal}function gtag(){dataLayer.push(arguments);}{/literal}
+    gtag('js', new Date());
+    gtag(
+      'config',
+      '{$gaAccountId|escape:'htmlall':'UTF-8'}',
+      {literal}{
+        'debug_mode':true
+        {if $gaAnonymizeEnabled}, 'anonymize_ip': true{/if}
+        {if $userId && !$backOffice}, 'user_id': '{$userId|escape:'htmlall':'UTF-8'}'{/if}
+        {if $backOffice && !$trackBackOffice}, 'non_interaction': true{/if}
+      }{/literal}
```

Comment:
```suggestion
      {ldelim}
        'debug_mode':true
        {if $gaAnonymizeEnabled}, 'anonymize_ip': true{/if}
        {if $userId && !$backOffice}, 'user_id': '{$userId|escape:'htmlall':'UTF-8'}'{/if}
        {if $backOffice && !$trackBackOffice}, 'non_interaction': true{/if}
      {rdelim}
```
This {literal} breaks the {if}'s inside it. So in the final code smarty outputs {if $.....}
![imagen](https://user-images.githubusercontent.com/10741347/177968335-102fbb3d-26a8-4dd1-a5c7-55d81df05b26.png)